safety/scan/command.py

```diff
@@ -175,6 +186,101 @@ def process_report(
     return report_url
 
 
+def filter_json_keys(json_string: str, keys: List[str]) -> str:
+    """
+    Filters the given JSON string by the specified top-level keys.
+
+    Args:
+        json_string (str): The JSON string to filter.
+        keys (List[str]): List of top-level keys to include in the output.
+
+    Returns:
+        str: A JSON string containing only the specified keys.
+    """
+    report_dict = json.loads(json_string)
+    filtered_data = {key: report_dict[key] for key in keys if key in report_dict}
+    return json.dumps(filtered_data, indent=4)
+
+
+def filter_valid_cves(vulnerabilities: List[Any]) -> List[Dict[str, Any]]:
+    """
+    Filters and returns valid CVE details from a list of vulnerabilities.
+
+    Args:
+        vulnerabilities (List[Any]): A list of vulnerabilities, which may include invalid data types.
+
+    Returns:
+        List[Dict[str, Any]]: A list of filtered CVE details that are either strings or dictionaries.
+    """
+    return [
```

Comment:
Question: When the data might be invalid? Do you have any examples of those?

Reply:
We don't have example that I know of, but this was just to make the code hardened